index.js

```diff
@@ -105,7 +105,7 @@ app.intent('send.message - collect.target', async conv => {
   const { convId } = users.length && await circuit.getDirectConversationWithUser(users[0].userId, true);
   const { message } = conv.contexts.input['sendmessage_data'].parameters;
   const name = users.length && users[0].displayName || convs[0].topic;
-  conv.ask(`<speak>Ready to send <break time="0.5s"/>${message}<break time="0.5s"/> to ${name}?</speak>`, new Suggestions('Yes', `No, don't send it`));
+  conv.ask(`<speak>Ready to send <break time='0.5s'/>${message}<break time='0.5s'/> to ${name}?</speak>`, new Suggestions('Yes', `No, don't send it`));
```

Comment:
Hmm, I'm not sure the quotes in break time ( and line 71) should be single quotes. I'm not too familiar with google assistant thought.